sdk/soap.class.php

```diff
@@ -177,7 +177,7 @@ public function setPerformanceLog($performancelog) {
         $this->performancelog = $performancelog;
     }
 
-    public function genUuid() {
+    public static function genUuid() {
```

Comment:
If this is supposed to be pulled in as a helper function it might be nicer as a trait instead or even a helper class. This SOAP class does a very specific thing and using it for different responsibilities is a code smell.